### PR TITLE
Fixes #87 - Connect daily_total script to the db

### LINKED ISSUE #87: Connect the sceduler daily_total script to the DB
Edit script to write to db rather than print to the console.

### PR DESCRIPTION
**Pull request by laghee**
I don't have a test for this yet because I need to figure out how to test a script in unittest. Maybe we want to pull out the act of writing to the database and make it a helper function? That might make it easier to mock.

🤷‍♀️ Let me know what you think, @karlcow ...

### CODE REVIEW

File: bin/daily_total.py [hunk 1]
```diff
@@ -63,11 +66,22 @@ def main():
             LOGGER.warning(msg)
             return
     # Format the data
-    data = "New issues on {yesterday}: {issue_count}".format(
-        yesterday=yesterday.isoformat(), issue_count=issue_count
-    )
-    # Log the data on the console
-    print(data)
+    yesterday = yesterday.isoformat()
```
Comment (karlcow) [inline bin/daily_total.py]: We can probably create a `iso_yesterday` at the top, just after
`yesterday = today - datetime.timedelta(days=1)`

and then reuse this anywhere the code uses `yesterday.isoformat()`, There are already 3 instances.

File: bin/daily_total.py [hunk 2]
```diff
@@ -63,11 +66,22 @@ def main():
...
+    count = issue_count
```
Comment (karlcow) [inline bin/daily_total.py]: Why do we need to rename the variable? We can reuse the same name everywhere.

**Comment by karlcow** [comment]
> I don't have a test for this yet because I need to figure out how to test a script in unittest. 

Currently the script is not very testable because it's all a big main function with a couple of functions, but it doesn't have to be. It's not that much of an issue for now. We started with a small script and it's ok. 

We can create more functions and associated tests with it. Basically you will not  "test a script" but you will be testing the functions in that script. We can create a new issue for this so we don't forget.

File: bin/daily_total.py (hunk 2 above)
Comment (laghee) [reply bin/daily_total.py]: I... don't know. I'm going to say "misguided futzing" and just take that out 😁.

**Comment by laghee** [comment]
Changes made, @karlcow!

**Pull request merged**